codewof/templates/programming/question.html

```diff
@@ -31,6 +31,9 @@ <h3>{{ question.title }}</h3>
             {% include "programming/question_components/editor-python.html" %}
         {% endif %}
 
+        <div id="indentation-warning" class="alert alert-danger d-none">
+            It looks like you have used tabs to indent your code. It is good Python style to use spaces.
```

Comment:
For the warning text, how about adding: `Please remove the tab character and try again.` to the end? It would make it clear that the user needs to remove them in order to get the question correct, rather than it just being a recommendation.